loadgen/demos/demo.py

```diff
@@ -0,0 +1,41 @@
+"""
+Python demo showing how to use the MLPerf Inference load generator bindings.
+"""
+
+from __future__ import print_function
+from absl import app
+import mlpi_loadgen
+import threading
+import time
+
+def LoadSamplesToRam(query_samples):
+  return
+
+def UnloadSamplesFromRam(query_samples):
+  return
+
+def ProcessQueryAsync(query_id, response_count):
+  time.sleep(.01)
+  r1 = mlpi_loadgen.QuerySampleResponse(0, 0);
+  r2 = mlpi_loadgen.QuerySampleResponse(0, 0);
+  r3 = mlpi_loadgen.QuerySampleResponse(0, 0);
+  r4 = mlpi_loadgen.QuerySampleResponse(0, 0);
+  responses = [r1, r2, r3, r4]
+  mlpi_loadgen.QueryComplete(query_id, responses)
+
+def IssueQuery(query_id, query_samples):
+  print(query_samples)
+  threading.Thread(target=ProcessQueryAsync, args=(query_id, len(query_samples))).start()
+
+def main(argv):
+  sut_name = "DemoSUT"
```

Comment:
I think those names don't do much - how about removing this argument?

Reply:
Done.